### DISCUSSION TITLE
SABCTools disabled: no correct version found! (Found v7.1.0, expecting v7.0.1) -Help

### DISCUSSION THREAD

**Discussion by alanialgor** [sabnzbd/sabnzbd#2614] [Q&A]
I have been upgrading my jails in FreeBSD to 13.1 and on of my services is Sabnzbd.  I upgraded sabnzbd from 3.6 to 4.0

now when i startup sabnzbd I get this error

SABCTools disabled: no correct version found! (Found v7.1.0, expecting v7.0.1)

hmmm.  I've tried a few things but to no avail.  does anyone have any suggestions?

thanks very much.
cheers

**Comment by thezoggy** [comment]
in sab we are pretty explicit of the version of sabctools to work with, with sab 4.0.x we want sabctools 7.0.x. with sab 4.1.x we want sabctools 7.1.x. this is all defined in our requirements.txt
it sounds like you just said install the latest version of x which now results in you having a version thats not supported for the verson you are trying to use with sab. 

either you need to install the correct dependencies or have someone else do it for ya (via package manager / docker maintainer)
if you are going to manage them yourself, you should do this for each time you install a new version of sab (from sab source dir):
```
python -m pip install --upgrade pip wheel
pip3 install --upgrade -r requirements.txt
pip3 install --upgrade -r builder/requirements.txt
```

tl;dr; you have the wrong version of sabctools installed, downgrade to the appropriate version for the version of sab you are running.
